src/db/sqlite/lrsql/sqlite/record.clj

```diff
@@ -16,6 +17,22 @@
 (u/def-hugsql-db-fns "lrsql/sqlite/sql/update.sql")
 (u/def-hugsql-db-fns "lrsql/sqlite/sql/delete.sql")
 
+;; Schema Update Helpers
+
+(defn- update-schema-simple
```

Comment:
Should have the `!` at the end of this mutating fn.

Reply:
indeed

Reply:
https://github.com/yetanalytics/lrsql/pull/207/commits/45d4791186a35b00ce79a0b0c1306cf8edb4ca72